noxfile.py

```diff
@@ -1,41 +1,92 @@
+import sys
+
 import nox
 
-nox.options.sessions = ["tests"]
-python_versions = ("3.10", "3.11", "3.12")
+nox.options.default_venv_backend = "uv"
 
+supported_python_versions = ("3.11", "3.12", "3.13")
 
-@nox.session
-def tests(session):
-    """
-    Run tests with pytest.
-    """
-    pytest_options = {}
-    session.install(".[tests]")
-    session.run("pytest", *pytest_options)
+maxpython = max(supported_python_versions)
+minpython = min(supported_python_versions)
+docpython = "3.13"
+
+current_python = f"{sys.version_info.major}.{sys.version_info.minor}"
+
+nox.options.sessions = [f"tests-{current_python}(all)"]
```

Comment:
The line will make it so that when we run `nox` without arguments on Python 3.13, the session that will be run is `tests-3.13(all)`.